### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JohannesLichtenberger**
Please complete the following checklist if your PR is adding new link to the list:

- [x] I've read [contributing guidelines](https://github.com/MunGell/awesome-for-beginners/blob/master/CONTRIBUTING.md)
- [x] This PR does not introduce duplicates to the list
- [x] The link is added at the bottom of the list category
- [x] Suggested repository is maintained, have active community, is able to mentor new contributors and have issues with the suggested label
- [x] The link I add follows the suggested pattern:

```
- [Repository Name](link-to-repository-label) _(label: beginner-friendly-label-in-the-repository)_ <br> Description
```

Example link formatting:

```
- [awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners/labels/good-first-contribution) _(label: good-first-contribution)_ <br> A list of awesome beginners-friendly projects.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +164,8 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Video Hub App](https://github.com/whyboris/Video-Hub-App/labels/good%20first%20issue) _(label: good first issue)_ <br> Angular & Electron app for browsing and searching videos on your PC.
 - [Ancient Beast](https://github.com/FreezingMoon/AncientBeast/labels/easy) _(label: easy)_ <br> Turn based strategy game where you 3d print a squad of creatures with unique abilities in order to defeat your enemies.
 - [Binari](https://github.com/BrandonArmand/Binari/labels/up%20for%20grabs) _(label: up for grabs)_ <br> Interactive code editor with a live binary tree visual designed to teach new developers the fundementals of dynamic programming.
+- [SirixDB](https://github.com/sirixdb/sirix-svelte-front-end) _(label: good first issue)_ <br>
```
Comment (MunGell) [inline README.md]: Hi @JohannesLichtenberger 

Thanks for the PR, would you be able to change this to full URL to the label?

Comment (JohannesLichtenberger) [reply README.md]: Done :-)

**Comment by MunGell** [comment]
stalebot, still relevant

File: README.md (hunk 1 above)
Comment (JohannesLichtenberger) [reply README.md]: Oh :-D

Hopefully now... we'll add many issues in the coming days (still brand new ;-)).

**Pull request merged**